app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,145 @@
+<p>Appwrite uses email and SMS messages to communicate with users to perform authentication actions. Emails and SMS messages can be customized to fit your app's design and voice.</p>
+
+<p>Each Appwrite project can have its own set of unique templates. Templates also support localization, every template can be written in multiple languages and served depending on the [TODO: @dlohani]</p>
```

Comment:
not sure how this should go yet, for now we could just keep up to. We don't yet have method to set locale in projects?

Reply:
There is `client.setLocale` method that set's the locale header `X-Appwrite-Locale`, which then determines which local to use.